src/components/InlineTooltip.js

```diff
@@ -77,10 +82,11 @@ var InlineTooltip = React.createClass({
         var cn = this.className;
         /* jshint ignore:start */
         return <div className={cn()}>
-            <Icon
+            {this.props.type == 'icon' && <Icon
```

Comment:
Please, use === instead of ==
Also, it better to change this and following conditions in some different way. Now if the condition is false, it puts null into the DOM.